<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connection</class>
 <widget class="QWidget" name="Connection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="hostLabel">
     <property name="text">
      <string>Hostname/IP:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="hostInput"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="portLabel">
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="portInput"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Nickname:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="nameInput"/>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="okBtn">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="cancelBtn">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
